apps/greenhouse-management/package.json

```diff
@@ -101,4 +102,4 @@
     }
   },
   "appPreview": true
-}
\ No newline at end of file
+}
```

Comment:
It looks here like I deleted the last empty line - but my VS Code and Prettier seems to stop that. Its literally impossible to change here anything for me.